src/utils/auth.ts

```diff
@@ -13,19 +13,20 @@ export default class AuthUtils {
    *
    * @param keycloak keycloak instance
    */
-  public static keycloakInit = (keycloak: KeycloakInstance) => {
-    return new Promise<boolean>((resolve, reject) =>
-      keycloak.init({ onLoad: "login-required", checkLoginIframe: false })
+  public static keycloakInit = (keycloak: Keycloak) => {
+    return new Promise<boolean>((resolve, reject) => {
+      keycloak.init({ onLoad: "check-sso", checkLoginIframe: false })
         .then(resolve)
-        .catch(reject));
+        .catch(reject);
+    });
   };
 
   /**
    * Loads user profile from Keycloak
    *
    * @param keycloak keycloak instance
    */
-  public static loadUserProfile = (keycloak: KeycloakInstance) => {
+  public static loadUserProfile = (keycloak: Keycloak) => {
```

Comment:
Same here, .loadUserProfile is also promise-like so no need for this kind of promise wrapper.